client/me/help/contact-form-notice/live-chat-closure.jsx

```diff
@@ -13,7 +13,10 @@ import ContactFormNotice from 'me/help/contact-form-notice/index';
 import { useLocalizedMoment } from 'components/localized-moment';
 
 const DATE_FORMAT = 'LLL';
-const translate = useTranslate();
+
+export const easterHolidayName = translate( 'Easter', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **145** times:
`translate( 'Easter' )` *ES Score: 17*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.